### PR TITLE
copy url to admin creds secret

### PR DESCRIPTION
**Pull request by philbrookes**
some stuff in here on realm reconciliation, ignore for now.

### CODE REVIEW

File: pkg/keycloak/keycloak.go [hunk 1]
```diff
@@ -147,13 +148,37 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
...
+				return errors.Wrap(err, "error reading admin credentials")
+			}
+
+			adminCreds, err := h.k8sClient.CoreV1().Secrets(namespace).Get(kc.Spec.AdminCredentials, metav1.GetOptions{})
+			if err != nil {
+				return errors.Wrap(err, "failed to get the secret for the admin credentials")
+			}
+
+			adminCreds.StringData = map[string]string{}
+			adminCreds.StringData["ADMIN_USERNAME"] = string(secretList.Items[0].Data["user_name"])
```
Comment (dimitraz) [inline pkg/keycloak/keycloak.go:163]: Do you think we can rely on the fact that the keycloak secret is always the first item in the list?

Comment (philbrookes) [reply pkg/keycloak/keycloak.go:163]: My logic here, is that if there's more than one secret with this service's id, then it's most likely a duplicate and the earliest one is probably the correct one.

Comment (dimitraz) [reply pkg/keycloak/keycloak.go:163]: Oh yeah I understand

Comment (dimitraz) [reply pkg/keycloak/keycloak.go:163]: But I think the postgres secret is labeled with this service id as well

Comment (philbrookes) [reply pkg/keycloak/keycloak.go:163]: yes ok, so I've added a serviceType to the secret:
https://github.com/aerogearcatalog/keycloak-apb/commit/96d59b10b2d14bc5ef3763adf4c3df143514ae09

and updated this to look for that as well.

Comment (dimitraz) [reply pkg/keycloak/keycloak.go:163]: :+1:

**Review by dimitraz — APPROVED** [review]
:+1:

**Pull request merged**